packages/serverpod/lib/src/server/message_central.dart

```diff
@@ -33,15 +35,17 @@ class MessageCentral {
       );
       var data =
           Serverpod.instance.serializationManager.encodeWithType(message);
-      Serverpod.instance.redisController!.publish(channelName, data);
+      return await Serverpod.instance.redisController!
+          .publish(channelName, data);
```

Comment:
There should be a try/catch here that returns `false` if `publish` throws an exception

Reply:
Also, `!` should be changed to `?`, but if `await Serverpod.instance.redisController` returns `null`, then `false` should be returned.

Reply:
Good catch! Will fix that up as well 👍

Reply:
This method now throws a state error if the Redis controller is not configured.

The try-catch for the internal publish is inside of the `publish` method so we don't need that on the outside.